lib/sign_in_page.dart

```diff
@@ -9,29 +11,136 @@ class SignInPage extends StatelessWidget {
+                ),
+              ),
+              Row(
+                mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                children: [
+                  Padding(
+                    padding: EdgeInsets.only(right: 100),
+                    child: TextButton(
+                      child: Text(
+                        'Forgot password?',
+                        style: TextStyle(
+                          fontFamily: 'SpaceGrotesk',
+                          fontWeight: FontWeight.w600,
+                        ),
+                      ),
+                      onPressed: () {},
+                    ),
+                  ),
+                  TextButton(
+                    child: Text(
+                      'Create account',
+                      style: TextStyle(
+                        fontFamily: 'SpaceGrotesk',
+                        fontWeight: FontWeight.w600,
+                      ),
+                    ),
+                    onPressed: () {},
+                  )
+                ],
+              ),
+              Padding(
+                padding: EdgeInsets.only(top: 10, left: 300),
+                child: Container(
+                  decoration: BoxDecoration(
+                    shape: BoxShape.circle,
+                    color: Color(0xFFF4603D),
+                    boxShadow: [
+                      BoxShadow(
+                        color: Colors.grey.withOpacity(.5),
+                        spreadRadius: 2,
+                        blurRadius: 8,
+                        offset: Offset(0, 2),
+                      ),
+                    ],
+                  ),
+                  child: IconButton(
+                    icon: Icon(Icons.send),
+                    iconSize: 20,
+                    color: Colors.white,
+                    onPressed: () {
+                      context.read<AuthenticationService>().signIn(
+                            emailController.text.trim(),
+                            passwordController.text.trim(),
+                          );
+                    },
+                  ),
+                ),
+              ),
+              Container(
+                padding: EdgeInsets.only(top: 15),
+                width: 360,
+                child: ElevatedButton.icon(
+                  icon: Icon(CustomIcons.google),
+                  onPressed: () {},
+                  label: Text('Sign in with Google'),
+                ),
+              ),
+            ],
           ),
-          TextField(
-            controller: passwordController,
-            decoration: InputDecoration(
-              labelText: "Password",
+          Align(
+            alignment: Alignment.bottomCenter,
+            child: Container(
+              height: 200,
+              alignment: Alignment.bottomCenter,
+              child: SvgPicture.asset(
+                'assets/images/mountain.svg',
+                alignment: Alignment.bottomCenter,
+              ),
```

Comment:
When using keyboard as an input, the mountain svg renders over everything else. Might be a little counter-intuitive, but the last widgets of `Stack children` is rendered over top of the other stuff in `Stack` (in other words, the stack paints its children in order with the first child being at the bottom). So just moving the highlighted lines of code to the top of `Stack children` array would allow the actual login widgets to render over the mountains svg.

Note that this does make an overflow render error more visible (you can see some of the black and yellow stripes in the screenshot already), but that was there before and fixing that is in another comment of this PR.

![signal-2021-07-12-210338](https://user-images.githubusercontent.com/7365763/125374467-117c0d80-e355-11eb-981d-c05188df9a2b.jpeg)